fcs/download-paket.ps1

```diff
@@ -1,13 +1,15 @@
-$paketurl="https://github.com/fsprojects/Paket/releases/download/5.201.1/paket.exe"
+$paketurl="https://github.com/fsprojects/Paket/releases/download/5.210.1/paket.exe"
 $paketdir = Join-Path $PSScriptRoot ".paket"
 $paketpath = Join-Path $paketdir "paket.exe"
 
 # Enable TLS 1.2 and TLS 1.1 as Security Protocols
 [Net.ServicePointManager]::SecurityProtocol = `
     [Net.SecurityProtocolType]::Tls12,
     [Net.SecurityProtocolType]::Tls11;
-    
+
 if (-not (Test-Path "$paketpath")) {
-    mkdir "$paketdir"
+    if (-not (Test-Path "$paketdir")) {
```

Comment:
Isn't this check already done above?

Reply:
Unfortunately not, `$paketpath` is tested, but this includes the filename `paket.exe`. See description of the bug. If the directory `$paketdir` exists but not the file then the build failes.

Reply:
Makes sense.